build.cmd

```diff
@@ -5,4 +5,7 @@ if errorlevel 1 (
   exit /b %errorlevel%
 )
 
+set MSBUILD=packages\build\RoslynTools.MSBuild\tools\msbuild
```

Comment:
Maybe we could set that in the FAKE script.

Reply:
sure, that may also fix unix, which currently fails (because it still runs the bundled msbuild).

How would you set it inside Fake? Just set the env vars? or is there some config switch?

Reply:
Good question. I think in FAKE 4 the way would be to change the mutable field with the path.
But yeah I don't know how a good FAKE API for this would look like...